turborepo-tests/helpers/install_deps.sh

```diff
@@ -3,15 +3,15 @@
 set -eo pipefail
 
 # TODO: Should we default to pnpm here?
-PACKAGE_MANAGER=${1-npm}
-
-# If a lock file already exists, we will exit. Some fixtures already have the lock
-# file. The caller is responsible for deleting this before calling this script.
-# TODO: we should make the fixtures consistent in either having or not having lockfiles.
-if [[ -f "package-lock.json" || -f "pnpm-lock.yaml" || -f "yarn.lock" ]]; then
-  exit 0
+PACKAGE_MANAGER="npm"
```

Comment:
We could probably change this default, but it would be a whole thing for integration tests. We need to audit the tests we have and probably partition them, some that run against all package managers, and some that use the default (which can likely be changed to pnpm)